docs/src/app/components/prop-type-description.css

```diff
@@ -41,3 +41,11 @@
   font-size: 95%;
   min-width: 250px;
 }
+
+.propTypeDescription  .requiredProp {
```

Comment:
> That's a descendent selector

Sorry, wrong wording. Can we avoid descendent selector?
I think it's better to be as close as possible to the inline style approach.